src/responseBuilderHandler.js

```diff
@@ -2,15 +2,16 @@
 
 var ResponseBuilder = require('./ResponseBuilder');
 
-module.exports = function(promiseReturningHandlerFn, request, cb) {
+module.exports = function(promiseReturningHandlerFn, request, cb, CustomRespBuilderClass) {
    promiseReturningHandlerFn()
       .then(function(respBuilder) {
          // eslint-disable-next-line no-console
          console.log('completed with %s millis left', request.getContext().getRemainingTimeInMillis());
          cb(undefined, respBuilder.toResponse(request));
       })
       .catch(function(err) {
-         var respBuilder = new ResponseBuilder(request).error();
+         var RB = CustomRespBuilderClass || ResponseBuilder,
+             respBuilder = new RB(request).error();
```

Comment:
Why is `request` being passed into the constructor here? Is it in case any future implementation of `ResponseBuilder` needs it? `ResponseBuilder` itself does not.